source/tutorials_wine.rst

```diff
@@ -413,28 +413,28 @@ Paste code into the file:
    :linenos:
    :emphasize-lines: 7-14,22
 
-    kind: ModelTraining
-    id: wine-tutorial
-    spec:
-      model:
-        name: wine
-        version: 1.0
-      toolchain: mlflow  # MLFlow training toolchain integration
-      entrypoint: main
-      workDir: mlflow/sklearn/wine  # MLproject location (in GitHub)
-      data:
-        - connName: wine-tutorial
-          localPath: mlflow/sklearn/wine/wine-quality.csv # wine-quality.csv file (on GCS)
-      hyperParameters:
-        alpha: "1.0"
-      resources:
-        limits:
-          cpu: 4024m
-          memory: 4024Mi
-        requests:
-          cpu: 2024m
-          memory: 2024Mi
-      vcsName: odahu-flow-tutorial
+   kind: ModelTraining
+   id: wine-tutorial
+   spec:
+     model:
+       name: wine
+       version: 1.0
+     toolchain: mlflow  # MLFlow training toolchain integration
+     entrypoint: main
+     workDir: mlflow/sklearn/wine  # MLproject location (in GitHub)
+     data:
+       - connName: wine-tutorial
+         localPath: data-totorial/wine-quality.csv # wine-quality.csv file (on GCS)
```

Comment:
I made this change because in our tutorial is the instruction:

![image](https://user-images.githubusercontent.com/49116723/73138036-d99cdf80-4066-11ea-8b05-0a099a708899.png)

Reply:
remotePath - file in a GCS bucket
localPath - file in a training container
[documentation](https://docs.odahu.org/ref_trainings.html#training-data)

Reply:
localPath: data-totorial/wine-quality.csv # wine-quality.csv file (on GCS)

Then we should change:
localPath: mlflow/sklearn/wine/wine-quality.csv # wine-quality.csv file <s>(on GCS)</s>

**and change description to 446 line:**
- line 12: ``localPath`` relative path of the data file at the training (docker) container where data were put, in our case, from gitHub repository.